scripts/audio.js

```diff
@@ -94,6 +99,14 @@ H5P.Audio = (function ($) {
       audioButton.removeClass(PAUSE_BUTTON).addClass(PLAY_BUTTON_PAUSED);
     });
 
+    // Prevent play/pause interaction on button when disabled.
+    audioButton.get(0).addEventListener('keydown', function (event) {
+      event = event || window.event;
+      if (!self.isEnabled() && (event.keyCode === 13 || event.keyCode === 32)) {
+        event.preventDefault();
+      }
+    }, false);
```

Comment:
I don't understand why we need this? This keydown results in a click event, right? And in the click event handler we are already checking for isEnabled...

Reply:
You're right, redundant and removed.